### ISSUE TITLE
Failed to load the linter-solidity package

### ISSUE THREAD

**Issue by ZhangYu-xiu** [Latrasis/linter-solidity#38]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.51.0 x64
**Electron**: 5.0.13
**OS**: Mac OS X 10.15.6
**Thrown From**: [linter-solidity](https://github.com/Latrasis/linter-solidity) package 0.5.0

### Stack Trace

Failed to load the linter-solidity package

```
At soljson.cwrap is not a function

TypeError: soljson.cwrap is not a function
    at setupMethods (/packages/linter-solidity/node_modules/solc/wrapper.js:9:29)
    at /packages/linter-solidity/node_modules/solc/index.js:3:18)
    at /packages/linter-solidity/node_modules/solc/index.js:5:3)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:147498)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:151046)
    at Module.load (internal/modules/cjs/loader.js:701:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:633:12)
    at Function.Module._load (internal/modules/cjs/loader.js:625:3)
    at Module.require (/app.asar/static/index.js:72:46)
    at require (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:146814)
    at /packages/linter-solidity/lib/linter.js:4:20)
    at /packages/linter-solidity/lib/linter.js:54:3)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:147498)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:151046)
    at Module.load (internal/modules/cjs/loader.js:701:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:633:12)
    at Function.Module._load (internal/modules/cjs/loader.js:625:3)
    at Module.require (/app.asar/static/index.js:72:46)
    at require (internal/modules/cjs/helpers.js:14:16)
    at customRequire (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:1:695414)
    at Package.requireMainModule (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:3196139)
    at /Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:3182679
    at Package.measure (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:3181511)
    at Package.load (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:3182252)
    at PackageManager.loadAvailablePackage (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:381611)
    at /Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:380034
    at Config.transact (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:332004)
    at PackageManager.loadPackages (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:379972)
    at /Applications/Atom.app/Contents/Resources/app/static/<embedded>:1:716025
    at async Promise.all (index 0)
```

### Commands

```
```

### Non-Core Packages

```
autocomplete-solidity 0.3.4 
busy-signal 2.0.1 
intentions 1.1.5 
language-ethereum 0.3.9 
linter 3.1.1 
linter-solidity 0.5.0 
linter-solium 0.0.5 
linter-ui-default 2.1.2 
```

**Comment by Devilla** [comment]
[Enter steps to reproduce:]

1. Install and Enable linter-solidity 0.5.0 
2. Restart the IDE i.e. **Atom**: 1.52.0 x64

**Atom**: 1.52.0 x64
**Electron**: 6.1.12
**OS**: Ubuntu 20.04.1
**Thrown From**: [linter-solidity](https://github.com/Latrasis/linter-solidity) package 0.5.0


### Stack Trace

Failed to load the linter-solidity package

```
At soljson.cwrap is not a function

TypeError: soljson.cwrap is not a function
    at setupMethods (/packages/linter-solidity/node_modules/solc/wrapper.js:9:29)
    at /packages/linter-solidity/node_modules/solc/index.js:3:18)
    at /packages/linter-solidity/node_modules/solc/index.js:5:3)
    at Module.get_Module._compile (/usr/share/atom/resources/app/static/<embedded>:11:147498)
    at Object.value [as .js] (/usr/share/atom/resources/app/static/<embedded>:11:151046)
    at Module.load (internal/modules/cjs/loader.js:645:32)
    at Function.Module._load (internal/modules/cjs/loader.js:560:12)
    at Module.require (/app.asar/static/index.js:72:46)
    at require (/usr/share/atom/resources/app/static/<embedded>:11:146814)
    at /packages/linter-solidity/lib/linter.js:4:20)
    at /packages/linter-solidity/lib/linter.js:54:3)
    at Module.get_Module._compile (/usr/share/atom/resources/app/static/<embedded>:11:147498)
    at Object.value [as .js] (/usr/share/atom/resources/app/static/<embedded>:11:151046)
    at Module.load (internal/modules/cjs/loader.js:645:32)
    at Function.Module._load (internal/modules/cjs/loader.js:560:12)
    at Module.require (/app.asar/static/index.js:72:46)
    at require (internal/modules/cjs/helpers.js:16:16)
    at customRequire (/usr/share/atom/resources/app/static/<embedded>:1:695477)
    at Package.requireMainModule (/usr/share/atom/resources/app/static/<embedded>:11:3196655)
    at /usr/share/atom/resources/app/static/<embedded>:11:3183195
    at Package.measure (/usr/share/atom/resources/app/static/<embedded>:11:3181995)
    at Package.load (/usr/share/atom/resources/app/static/<embedded>:11:3182768)
    at PackageManager.loadAvailablePackage (/usr/share/atom/resources/app/static/<embedded>:11:381794)
    at /usr/share/atom/resources/app/static/<embedded>:11:380217
    at Config.transact (/usr/share/atom/resources/app/static/<embedded>:11:332187)
    at PackageManager.loadPackages (/usr/share/atom/resources/app/static/<embedded>:11:380155)
    at /usr/share/atom/resources/app/static/<embedded>:1:716064
    at async Promise.all (index 0)
    at AtomEnvironment.startEditorWindow (/usr/share/atom/resources/app/static/<embedded>:1:718103)
```

### Commands

```
     -1:09.5.0 Code-Time:process-keystrokes-data (atom-workspace.workspace.scrollbars-visible-always.theme-one-dark-syntax.theme-one-dark-ui)
  2x -0:46.8.0 Code-Time:refresh-code-time-metrics (atom-workspace.workspace.scrollbars-visible-always.theme-one-dark-syntax.theme-one-dark-ui)
  2x -0:37.8.0 Code-Time:process-keystrokes-data (atom-workspace.workspace.scrollbars-visible-always.theme-one-dark-syntax.theme-one-dark-ui)
     -0:27.4.0 Music-Time:refresh-treeview (atom-workspace.workspace.scrollbars-visible-always.theme-one-dark-syntax.theme-one-dark-ui)
     -0:15.3.0 Code-Time:refresh-code-time-metrics (atom-workspace.workspace.scrollbars-visible-always.theme-one-dark-syntax.theme-one-dark-ui)
```

### Non-Core Packages

```
atom-beautify 0.33.4 
code-time 4.3.3 
linter-solidity 0.5.0 
music-time 2.0.13 
teletype 0.13.4 
```

**Comment by Hamish-Gibson** [comment]
I've tried the above solution and I installed the necessary other packages yet this still doesn't work. Can you provide another solution?

**Comment by Devilla** [comment]
It's not a solution it's the issue I also faced.

**Comment by Latrasis** [comment]
Apologies again for no feedback since the last few years, I haven't had the chance to use Atom for a while, but I'll see if I can help. Any PR's are welcome.

**Comment by PaarthAazaad-Bhaarati** [comment]
Do you have any solution to that.....

**Comment by FanM** [comment]
This is already fixed by #31. Or you can remove that solc import from /packages/linter-solidity/lib/linter.js and restart Atom. @Latrasis Could you release a new version to apm to avoid people hitting this?
